cfgov/cfgov/settings/local.py

```diff
@@ -0,0 +1,14 @@
+from .base import *
+import v1
+
+DEBUG = True
+
+ALLOWED_HOSTS = ['*']
+
+STATIC_URL = '/static/'
+
+SHEER_SITES = [os.path.dirname(v1.__file__) + '/jinja2/v1']
```

Comment:
Along with the `/docs/` url pattern, this needs to specify docs as a sheer site.